news/10504.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fix a crash when installing an editable requirement using the legacy resolver
```

Comment:
Did this ever make it into a release? If not, let’s drop this entry, since there’s no user facing behaviour changes here.

Reply:
Honestly I’m not sure when the breaking change was made, the refactoring dragged for quite a while.

Reply:
I filed https://github.com/pypa/pip/pull/10523 to remove the news file, as the bug as not present in 21.2